packages/keymaps/src/browser/keybindings-widget.tsx

```diff
@@ -492,8 +496,9 @@ export class KeybindingWidget extends ReactWidget {
      */
     protected getCommandLabel(command: Command): string {
         if (command.label) {
+            const label = LocalizationInfo.localize(command.label, this.localizationService);
             // Prefix the command label with the category if it exists, else return the simple label.
-            return command.category ? `${command.category}: ${command.label}` : command.label;
+            return command.category ? `${command.category}: ${label}` : label;
```

Comment:
It looks like command categories are also available for localization in VSCode:
![image](https://user-images.githubusercontent.com/62660806/120397918-d4a50b80-c2fe-11eb-9e22-b8cb376212ad.png)

Would it be possible to add those to the PR conveniently?

Reply:
Totally forgot about those. Done (for the 'File' command entries at least)

Also, you reminded me that vscode shows the original text under the localized text. Added this feature to the planned features for this PR.